langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -222,6 +306,10 @@ public type JsonFloat ()|boolean|string|float|JsonFloat[]|map<JsonFloat>;
 #
 # Returns an error if the string cannot be parsed.
 #
+# ```ballerina
+# "[12, true, 123.4, \"hello\"]".fromJsonFloatString() ⇒ [12.0,true,123.4,"hello"]
+# ```
```

Comment:
An error case?

Reply:
Check other places as well. `fromJsonDecimalString` is also missing an error case